fe2-android/app/src/main/java/com/github/dedis/student20_pop/home/HomeActivity.java

```diff
@@ -296,10 +308,15 @@ private void setupSeedWalletFragment() {
     }
   }
 
-  private void openLaoDetails(String laoId) {
+  private void openLaoDetails(String laoId, boolean openLaoDetail) { //change to enum??
```

Comment:
You can have two different methods, `openLaoDetails` and `openContentWallet`, that call this method with the respective true or false. If you do that you'll need to generalise the name of this method, you can name it `openLaoDetailActivity` for example